packages/react-widgets/src/less/variables.less

```diff
@@ -159,9 +159,9 @@
 @multiselect-tag-border:          @btn-border;
 @multiselect-tag-bg:              @gray-lighter;
 @multiselect-tag-height-pt:       1 - (@multiselect-gutter-pt * 2);
-@multiselect-tag-gutter-fallback: @input-height * @multiselect-gutter-pt;
+@multiselect-tag-gutter-fallback: ~"calc(@{input-height} * @{multiselect-gutter-pt})";
 @multiselect-tag-gutter:          ~"calc(@{multiselect-tag-gutter-fallback} - @{input-border-width})";
-@multiselect-tag-height:          @input-height * @multiselect-tag-height-pt;
+@multiselect-tag-height:          ~"calc(@{input-height} * @{multiselect-tag-height-pt})";
```

Comment:
why is this needed? calc is a runtime calculation which  may not be supported by our lowest supported browsers

Reply:
It doesn't compile correctly when compiled to sass, in the last PR another person experience the same problem. 

Calc is needed because the input-height and tag-height are two different types that are not compatible without calc

Reply:
sorry, I realize why this solves the problem, I'm saying it's solving it in a backwards incompatible way, so we should look for a different approach

Reply:
The other approach is to maintain SASS files separately. Are the LESS files breaking for you on import? Or the compile to CSS? This fixes importing the SASS files, I don't see breaks in this but I could be wrong. Could you share what errors you are seeing?

Reply:
Sorry I miss understood this message.

The other approach is to maintain SASS files separately which I would prefer personally because LESS and SASS compile differently under the hood. SASS using a C/C++ compiler, that's why I prefer it over LESS. But this is a great hot fix

Reply:
Also calc is already being used in that file so it doesn't change the current browser support

Reply:
Sorry I miss understood your message.

Calc is currently being used in the project already which will not change browser support

Reply:
this is still relevant, please remove the added calcs since they are not needed, thanks!

Reply:
Calc is absolutely needed if you have a better solution I am open for suggestions. They are already present in the code base so browser compatibility should not be an issue. There will be two additional calc methods where they is already calcs in the codebase. It's been year that this PR is open and people are still having issues. For two simple updates, this is my personal solution. I hope others will use it in their code bases that prefer custom SASS styling.

Reply:
hi @laynef - your fix address the compile issue but I'm noticing a strange padding issue with multiselect field. attaching two screenshots below for reference, let me know if these aren't related. thanks!

<img width="608" alt="Screen Shot 2020-05-08 at 5 51 12 AM" src="https://user-images.githubusercontent.com/2752842/81357247-bf0be500-90f0-11ea-9ce7-529785930dcd.png">
<img width="595" alt="Screen Shot 2020-05-08 at 5 51 17 AM" src="https://user-images.githubusercontent.com/2752842/81357250-c16e3f00-90f0-11ea-8660-4136d68a8a21.png">

Reply:
This is a valid concern and I would love to find a solution that fixes this however I'd had too many issues with this project and the wait time for all contributions have been too long. I  personally don't use this package anymore. Closing this PR gives me solace.

Reply:
guy, not to nitpick here but you also have many multi month delays in responding to my questions and feedback here.